web/templates/personal_menu.hbs

```diff
@@ -84,6 +84,33 @@
                         {{/if}}
                     </ul>
                 </li>
+                <li class="navbar-dropdown-menu-outer-list-item">
+                    <ul class="navbar-dropdown-menu-inner-list">
+                        <li class="navbar-dropdown-menu-inner-list-item">
+                            <div class="theme-switcher">
+                                <input type="radio" id="select-automatic-theme" class="theme-option"  name="theme-select" data-theme-code="1" {{#if (eq user_color_scheme 1)}}checked{{/if}} />
```

Comment:
Rather than hardcoding 1/2/3 here, I think you can pass through a `settings_config.color_scheme_values` structure, or something computed from it, so that this template code is readable as to what each value means.

Reply:
Followed your feedback and removed the hard-coded values from the template, thanks!